src/index.js

```diff
@@ -14,26 +14,27 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-const db = require('./interfaces/db');
 const eth = require('./interfaces/eth');
 const net = require('./interfaces/net');
 const parity = require('./interfaces/parity');
 const personal = require('./interfaces/personal');
-const privateMod = require('./interfaces/private');
+const private = require('./interfaces/private');
 const shh = require('./interfaces/shh');
 const signer = require('./interfaces/signer');
 const trace = require('./interfaces/trace');
 const web3 = require('./interfaces/web3');
+const secretstore = require('./interfaces/secretstore');
 
 module.exports = {
-  db,
   eth,
   parity,
   net,
   personal,
-  private: privateMod,
-  shh,
+  private,
+// TODO review all shh function in rust to match with JSON RPC functions
+//  shh,
   signer,
   trace,
-  web3
+  web3,
+  secretstore
```

Comment:
we have this implicit convention in JS code to order everything alphabetically, if no other natural ordering imposes itself.